plugins/outputs/influxdb_v3/influxdb_v3.go

```diff
@@ -0,0 +1,220 @@
+//go:generate ../../../tools/readme_config_includer/generator
+package influxdb_v3
+
+import (
+	"context"
+	"crypto/tls"
+	_ "embed"
+	"errors"
+	"fmt"
+	"math/rand"
+	"net"
+	"net/url"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/config"
+	"github.com/influxdata/telegraf/internal"
+	"github.com/influxdata/telegraf/plugins/common/ratelimiter"
+	commontls "github.com/influxdata/telegraf/plugins/common/tls"
+	"github.com/influxdata/telegraf/plugins/outputs"
+	"github.com/influxdata/telegraf/plugins/serializers/influx"
+)
+
+//go:embed sample.conf
+var sampleConfig string
+
+type InfluxDB struct {
+	URLs               []string          `toml:"urls"`
+	LocalAddr          string            `toml:"local_address"`
+	Token              config.Secret     `toml:"token"`
+	Database           string            `toml:"database"`
+	DatabaseTag        string            `toml:"database_tag"`
+	ExcludeDatabaseTag bool              `toml:"exclude_database_tag"`
+	Timeout            config.Duration   `toml:"timeout"`
+	HTTPHeaders        map[string]string `toml:"http_headers"`
+	HTTPProxy          string            `toml:"http_proxy"`
+	UserAgent          string            `toml:"user_agent"`
+	ContentEncoding    string            `toml:"content_encoding"`
```

Comment:
Can't most just come from common HTTPClientConfig?

Reply:
Good point. Will look into this.